packages/pytest-ipywidgets/pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[build-system]
+requires = ["hatchling==1.22.2"]
```

Comment:
Why fix `hatchling` version for the new packages, while it isn't fixed in the `solara-ui` (formerly `solara`)?

Reply:
Yes, lets do it consistently, and lock it, so we don't get surprises with releases. I think we can bump if when we need a new feature of hatch